testvectors/tf-rmm/cca_platform.pem

```diff
@@ -0,0 +1,6 @@
+-----BEGIN EC PRIVATE KEY-----
```

Comment:
Nice!
Can you please also add associated EC public Key. This will be used by me in TA provisioning.

Reply:
No need to stash it there, one can just:
```bash
openssl ec -in cca_platform.pem -pubout -out cca_platform.pub
```

Reply:
I  understand that user can search and use commands from internet to extract the corresponding public key.

My suggestion was more from a direction of convenience and consistency of information.
It is far more easy if we put tried and tested valid Private-Public Key pair in the repo for ease of use when working towards new fixes or enhancement.

Reply:
Whilst I am still unsure about the usefulness of this, I've added it.

Reply:
Thanks! It is and will be helpful, given the fact we have retained the key pair is a very convenient option!